### PR TITLE
Add centosah

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by ashcrow** [comment]
I asked @abutcher to take a gander.

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -93,6 +103,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     v.memory = 1024
     v.cpus = 1
     override.vm.box = box_name
+    if override_box_url
+      override.vm.box_url = override_box_url
+    end
```
Comment (abutcher) [inline vagrant/Vagrantfile]: Should this box_url override live under the libvirt config? I got a libvirt box with --provider=virtualbox

```
https://ci.centos.org/artifacts/sig-atomic/centos-continuous/images-alpha/cloud/latest/images/centos-atomic-host-7-vagrant-libvirt.box
```

Comment (abutcher) [inline vagrant/Vagrantfile]: I see a virtualbox image in the root so probably something strange happening locally.

**Comment by cooktheryan** [comment]
I hit this while attempting to test https://gist.github.com/cooktheryan/2facf392c7ad871df2b12997eb22d02a with virtualbox

File: vagrant/Vagrantfile (hunk 1 above)
Comment (cooktheryan) [inline vagrant/Vagrantfile]: https://gist.github.com/cooktheryan/7d2be1d3d2ff6262587b715acd7ed5ad had a similar vbox issue

**Comment by cgwalters** [comment]
Any comments on the fixup?  I know the notice may have been lost due to the github outage.

**Comment by cooktheryan** [comment]
@cgwalters if you can hook me up with a quick how to guide on performing the steps I would be glad to help validate. I attempted the other day but ran into an issue when it was attempting to run a network manager restart

**Comment by ashcrow** [comment]
@cooktheryan do you happen to have the output of the issue you ran into?

**Comment by cooktheryan** [comment]
@ashcrow @cgwalters https://gist.github.com/cooktheryan/6bb6b23364e00ea6851abe08511172f9

**Comment by cooktheryan** [comment]
```
-- Unit network.service has begun starting up.
Dec 09 19:30:12 master1.example.com network[12528]: Bringing up loopback interface:  Could not load file '/etc/sysconfig/network-scripts/ifcfg-lo'
Dec 09 19:30:12 master1.example.com network[12528]: Could not load file '/etc/sysconfig/network-scripts/ifcfg-lo'
Dec 09 19:30:12 master1.example.com network[12528]: Could not load file '/etc/sysconfig/network-scripts/ifcfg-lo'
Dec 09 19:30:13 master1.example.com network[12528]: Could not load file '/etc/sysconfig/network-scripts/ifcfg-lo'
Dec 09 19:30:13 master1.example.com network[12528]: [  OK  ]
Dec 09 19:30:13 master1.example.com network[12528]: Bringing up interface eth0:  Error: Connection activation failed: No suitable device found for this connection.
Dec 09 19:30:13 master1.example.com network[12528]: [FAILED]
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com network[12528]: RTNETLINK answers: File exists
Dec 09 19:30:13 master1.example.com systemd[1]: network.service: control process exited, code=exited status=1
Dec 09 19:30:13 master1.example.com systemd[1]: Failed to start LSB: Bring up/down networking.
```

**Comment by cgwalters** [comment]
Can you post the output of `atomic host status` and `ls -alZ /etc/sysconfig/network-scripts/ifcfg-lo` and look at `journalctl -b -p err` etc?

**Comment by cgwalters** [comment]
For me:

```
# atomic host status
State: idle
Deployments:
● centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/alpha
       Version: 7.2016.1024 (2016-12-09 21:17:00)
        Commit: f0894de40ef404555cf794d9d279eab03839738bf84ea6f9703c80ca79c2903b
        OSName: centos-atomic-host
# systemctl status network
● network.service - LSB: Bring up/down networking
   Loaded: loaded (/etc/rc.d/init.d/network; bad; vendor preset: disabled)
   Active: active (exited) since Mon 2016-12-12 20:56:58 UTC; 8min ago
     Docs: man:systemd-sysv-generator(8)
  Process: 881 ExecStart=/etc/rc.d/init.d/network start (code=exited, status=0/SUCCESS)

Dec 12 20:56:58 localhost.localdomain systemd[1]: Starting LSB: Bring up/down networking...
Dec 12 20:56:58 localhost.localdomain network[881]: Bringing up loopback interface:  [  OK  ]
Dec 12 20:56:58 localhost.localdomain network[881]: Bringing up interface eth0:  [  OK  ]
Dec 12 20:56:58 localhost.localdomain systemd[1]: Started LSB: Bring up/down networking.
```

**Comment by cooktheryan** [comment]
Got busy today will shoot you a log tomorrow

**Comment by cgwalters** [comment]
We actually shouldn't have `network.service` enabled; https://github.com/CentOS/sig-atomic-buildscripts/pull/197

**Comment by cooktheryan** [comment]
@cgwalters got a lot further during the installation process. I hit this wall on the admin box

```
==> admin1: Setting hostname...
==> admin1: Configuring and enabling network interfaces...
==> admin1: Installing SSHFS client...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!
dnf -y install fuse-sshfs
Stdout from the command:
bash: dnf: command not found
Stderr from the command:
[rcook@goku vagrant]$ rpm -qa | grep fuse-sshfs
fuse-sshfs-2.8-1.fc25.x86_64
```

**Comment by cgwalters** [comment]
You need `fuse-sshfs` on the VM too.  I've been doing `rpm-ostree install fuse-sshfs ansible pyOpenSSL` manually so far, at some point we'll do https://pagure.io/fedora-atomic/issue/38

**Comment by cgwalters** [comment]
I'd also like to look at migrating the ansible invocations into a container.

**Comment by cooktheryan** [comment]
```
[vagrant@admin1 ~]$ sudo rpm-ostree install ansible
Checking out tree f0894de... done

Downloading metadata: [======================================================================================================================================================================================] 100%
error: No package matches 'ansible'
/
```

**Comment by cgwalters** [comment]
You need to install EPEL too:
```
(cd / && curl https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm  | rpm2cpio | cpio -div)
```

**Comment by cooktheryan** [comment]
@cgwalters after installing those items and re-running the <DEPLOYMENT_TYPE=origin ORIGIN_OS=centos-ah vagrant up --provider=virtualbox> command but the one question I have is there a way to cloud-init those items or have an if script so that the items tasks that need to be done are completed?

**Comment by cgwalters** [comment]
Unfortunately there's no good way to do that right now because vagrant-sshfs injects its run before any provisioners are called.   It currently installs EPEL automatically which I think is a bit presumptuous; I filed https://github.com/dustymabe/vagrant-sshfs/issues/63

This does suck but I'm trying not to get bogged down by every problem in the the chain.  Maybe I can create a one-liner for this.

**Comment by cooktheryan** [comment]
@cgwalters did you still want to look at a 1 liner for this or should we move forward?

**Comment by ashcrow** [comment]
@cooktheryan I recommend moving forward. The one liner could be done as a later follow on if needed.

**Comment by cgwalters** [comment]
OK, so I baked in `fuse-sshfs` into the CAHC boxes by default.  This works a bit better now, can you retry?

**Comment by cooktheryan** [comment]
I got tied up today so I should be able to test this on monday

**Comment by cgwalters** [comment]
To be really seamless I need to finish porting to run Ansible from a container; see ⬆️

**Comment by cooktheryan** [comment]
@cgwalters if you want to resolve those conflicts ill test and merge

**Comment by ashcrow** [comment]
Is this still being pursued?

**Comment by ashcrow** [comment]
@cgwalters Is this still being pursued?

**Pull request closed without merging**